### ISSUE TITLE
Awkward redstone chain

### ISSUE THREAD

**Issue by Agrax** [pyure/infitech-prototype#470]
Redstone dust->compressor -> redstone plate -> compressor -> redstone block -> cutting saw - > redstone plate

The recipe should be as follows:

redstone dust-> compressor -> redstone block -> cutting saw - > redstone plate

Redstone plates twice in the chain? Gee

**Comment by pyure** [comment]
I don't understand.  If you'r trying to get a redstone plate (which is thelast thing you put in the chain) why arent you stopping after 3 steps when you have redstone plates?  Did you typo?

**Comment by Agrax** [comment]
The recipe chain NOW, *in the release 0.44* is:

```Redstone dust->in a compressor -> redstone plate -> in a compressor -> redstone block -> in a cutting saw - > redstone plate```


Why there is a redstone plate TWO times? Why do you have to compress redstone TWICE to get a block? This doesn't make the least amount of sense- neither to me, nor to Jason McRay. That's why we've came up with THIS chain, which makes a whole lot more of sense, because redstone plates don't appear in it two times:

```redstone dust-> in a compressor -> redstone block -> in a cutting saw - > redstone plate```

This way it takes less time, effort and energy to make redstone blocks which are used more frequently than a two-step process. 

My point beeing: the state in which the recipe is now doesn't make a whole lot of sense and should be fixed to make sense. Jason would agree, as he was the one to see this awkward thing, as he was making an ID battery, which requires redstone blocks.

Did i make myself clear enough this time?

**Comment by pyure** [comment]
No its still ambiguous because you say you're trying to get a block but you're ending at a redstone plate. After reading it several times I see that you're demonstrating the whole chain, but mentioning the plate maybe as an example.

The redstone block thing is already a known issue from weeks ago.  It was added because of a compressor/redstone conflicting recipe and known to be a sketchy and temporary solution.  In fact, I'm pretty sure yours conflicted.

If yours is (no longer?) a conflict, there's no problem putting it back in.

**Comment by Agrax** [comment]
The solution should have been:

```removing the redstone plate compression```

Redstone-> compressor-> redstone block

The way to get plates should be cutting the block: the same like it is for every other crystal-type resource

**Comment by Agrax** [comment]
https://github.com/pyure/infitech-prototype/blob/0fc53c8c3e38dca16ce1125ae849b3420575e951/scripts/gregtech.zs#L108

No wonders it conflicted

**Comment by Divran** [comment]
I also mentioned this on the discord a while back. Below is the complete chat log. 

I mentioned then that the way you suggested would be fine functionally, but that the IT2 way would be more consistent with other materials.
I think it makes more sense for the recipe to use the IT2 way, which is `redstone -> compressor=block -> cutting saw=plate`

```
[03/23/2019]
[12:23 AM] Divran: @pyure found a recipe conflict https://i.imgur.com/yTvxwoN.png
[12:24 AM] Divran: this makes it impossible to make redstone blocks before LV (you can still make them via fluid solidifier)
[1:11 AM] The Red: Is there any need for redstone blocks before LV?
[1:12 AM] Divran: I needed one for thaumcraft research
[1:12 AM] Divran: but its still a recipe conflict and it should be fixed regardless
[1:29 AM] pyure: nod agreed.  Weird thing is, I recall us fixing it before I think.  I wonder if its another breakage from GA
[1:43 AM] pyure: It looks like the redstone BLOCK is a IT3 recipe
[1:44 AM] pyure: Not sure how to replace it yet
[1:46 AM] pyure: @Divran would it be really annoying if I did
dustRedstone -> plateRedstone (already there)
and
plateRedstone * 9 -> blockRedstone
?
[1:46 AM] pyure: You'd have an extra step but it would remove the conflict (and sorta makes sense)
[3:31 PM] Divran: @pyure that would be fine imo, but it would not be consistent with other materials. if I remember ic2 correctly, to get redstone plate you would put a block in a cutting machine
[3:32 PM] Divran: this cutting machine recipe already exists in it3
[3:32 PM] Divran: and it should also exist for a bunch of other materials such as lapis
[3:33 PM] Divran: so you could change it the way you suggested above but then you would have to change all other similar materials the same way to maintain consistency
```

**Comment by Agrax** [comment]
#468 Here's my fix. Somewhere

**Comment by pyure** [comment]
Fixed in infitech-prototype-0.45
